Signal/src/ViewControllers/ConversationView/ConversationInputTextView.h

```diff
@@ -10,8 +10,6 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)didPasteAttachment:(SignalAttachment *_Nullable)attachment;
 
-- (void)textViewDidChangeLayout;
```

Comment:
We don't need to observe the layout of the input toolbar or text view anymore; it's redundant with and less useful than our new approach of observing layout changes to the collection view.